pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
No, it's specifically omitted because we are not building a new version of the SNI packages.  Instead we're reusing the RTM ones.